spec.html

```diff
@@ -34532,8 +34534,203 @@ <h2>Syntax</h2>
+                <ul>
+                  <li>
+                    `/` for ES regex literals, and for lexer scanning
+                  </li>
+                </ul>
+              </li>
+              <li>
+                Characters that should be escaped:
+                <ul>
+                  <li>
+                    `(` `)` reserved for future extensions, and for readability
+                  </li>
+                  <li>
+                    `{` `}` reserved for future extensions, and for readability
+                  </li>
+                  <li>
+                    `|` for consistent escaping anywhere in a character class (needed in string literals)
+                  </li>
+                </ul>
+              </li>
+            </ul>
+          </li>
+          <li>
+            FYI: Required escaping in character classes string literals:
+            <ul>
+              <li>
+                Characters that must be escaped (to make the syntax work):
+                <ul>
+                  <li>
+                    `{` `}` for end-of-string-literal, and the open brace for readability
+                  </li>
+                  <li>
+                    `\\` for escape sequences
+                  </li>
+                  <li>
+                    `|` for multiple-string literals
+                  </li>
+                </ul>
+              </li>
+              <li>
+                Characters that must be escaped (for one or another lexer):
+                <ul>
+                  <li>
+                    `[` `]` for lexer scanning
+                  </li>
+                  <li>
+                    `/` for ES regex literals, and for lexer scanning
+                  </li>
+                </ul>
+              </li>
+              <li>
+                Characters that should be escaped:
+                <ul>
+                  <li>
+                    `(` `)` reserved for future extensions, and for readability
+                  </li>
+                  <li>
+                    `-` for consistent escaping anywhere in a character class (needed in ranges)
+                  </li>
+                </ul>
+              </li>
+            </ul>
+          </li>
+        </ul>
+      </emu-note>
+      <emu-grammar type="definition">
+        ClassAllowEscaped :: one of
+          `&amp;` `-` `!` `#` `%` `,`
+          `:` `;` `&lt;` `=` `&gt;` `@`
+          `_` `&grave;` `~`
+      </emu-grammar>
+      <emu-note>
+        <ul style="background: yellow">
+          <li>
+            FYI: Allow escaping those characters in ClassReservedDouble that are not otherwise allowed via CharacterClassEscape.
+          </li>
+          <li>
+            Without this, one would have to put these characters into [nested classes].
+          </li>
+          <li>
+            Another option is to put all of these, and SyntaxCharacter, into ClassSyntaxCharacter, requiring them to be escaped. But that seems unfriendly.
```

Comment:
Closing this as well since this was actually considered originally. If I misread and you actually feel strongly that we need to change this, please file an issue.

Reply:
moved to https://github.com/tc39/proposal-regexp-set-notation/issues/59